charts/pulsar/templates/broker/function-worker-configfile-configmap.yaml

```diff
@@ -106,6 +106,15 @@ data:
     brokerClientTrustCertsFilePath: "/pulsar/certs/ca/ca.crt"
     {{- end }}
 
+    {{- if .Values.functions.skipTls }}
+    tlsAllowInsecureConnection: false
+    tlsEnableHostnameVerification: false
+    {{- end }}
+    
+    {{- if .Values.functions.RefreshCheckDurationSec }}
```

Comment:
Should the name starts with a lowercase letter?

Reply:
updated to .Values.functions.tlsCertRefreshCheckDurationSec